src/main/java/teachmyskill/task/servlet/CreateUserServlet.java

```diff
@@ -0,0 +1,39 @@
+package teachmyskill.task.servlet;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.sql.*;
+
+import static teachmyskill.task.database.Connect.getConnect;
+@WebServlet("/create")
+public class CreateUserServlet extends HttpServlet {
+
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        Connection connection = getConnect();
+        try {
+            PreparedStatement preparedStatement = connection.prepareStatement("INSERT INTO employees VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)");
+            preparedStatement.setInt(1, Integer.parseInt(req.getParameter("id_c")));

```

Comment:
Usually, user don't enter the ID. This field should be either generated with DB auto increment, or we set it manually on the backend